modules/gcp/k8s/main.tf

```diff
@@ -20,6 +20,13 @@ resource "google_container_cluster" "tsb" {
   # node pool and immediately delete it.
   remove_default_node_pool = true
   initial_node_count       = 1
+
+  resource_labels = {
+    name        = "${var.cluster_name}_tsb_sandbox_blue"
+    environment = "${var.name_prefix}_tsb"
+    owner       = var.owner
```

Comment:
so for AWS and Azure we have converged to Tetrate:Owner, but for GCP it is Owner label, can we do the same for GCP?

Reply:
That was my attempt, but google only allows lowercase (I had it in my first GCP commit, but had to revert that after errors)

https://cloud.google.com/resource-manager/docs/creating-managing-labels#requirements

> Keys and values can contain only lowercase letters, numeric characters, underscores, and dashes. All characters must use UTF-8 encoding, and international characters are allowed.

Reply:
ack, thanks for the context